src/main.rs

```diff
@@ -132,73 +154,22 @@ extern "C" fn sample_main() {
         // Wait for either a specific button push to exit the app
         // or an APDU command
         if let Event::Command(ins) = ui_menu_main(&mut comm) {
-            match handle_apdu(&mut comm, ins.into(), &mut tx_ctx) {
+            match handle_apdu(&mut comm, ins, &mut tx_ctx) {
                 Ok(()) => comm.reply_ok(),
                 Err(sw) => comm.reply(Reply::from(sw)),
```

Comment:
Here we can just write
```suggestion
                Err(sw) => comm.reply(sw),
```
I thought I already committed that in another PR.